### ISSUE TITLE
EventException on mvtp

### ISSUE THREAD

**Issue by SlimeDog** [RockinChaos/ItemJoin#179]
Spigot 1.13 latest build
ItemJoin 4.9-b134

This occurred when I executed /mvtp command to the spawn point in the current world (on this test server I have only a single world).

```
[09:15:02 ERROR]: Could not pass event PlayerMoveEvent to ItemJoin v4.9-SNAPSHOT
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:931) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.PacketPlayInFlying.a(SourceFile:126) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.PacketPlayInFlying$PacketPlayInLook.a(SourceFile:88) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.a(SourceFile:10) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_171]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_171]
        at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) [spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:884) [spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:411) [spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:819) [spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:717) [spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.NoSuchMethodError: com.sk89q.worldguard.bukkit.WorldGuardPlugin.getRegionManager(Lorg/bukkit/World;)Lcom/sk89q/worldguard/protection/managers/RegionManager;
        at me.RockinChaos.itemjoin.listeners.giveitems.RegionEnter.getWGSet(RegionEnter.java:224) ~[?:?]
        at me.RockinChaos.itemjoin.listeners.giveitems.RegionEnter.CheckInRegion(RegionEnter.java:207) ~[?:?]
        at me.RockinChaos.itemjoin.listeners.giveitems.RegionEnter.isInRegion(RegionEnter.java:187) ~[?:?]
        at me.RockinChaos.itemjoin.listeners.giveitems.RegionEnter.giveOnRegionEnter(RegionEnter.java:43) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_171]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_171]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_171]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_171]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304) ~[spigot-1.13.jar-2018-07-30-0857:git-Spigot-69774b3-8592585]
        ... 15 more
```

**Comment by RockinChaos** [comment]
Honestly, I think I am pretty much out of luck until WorldGuard officially updates. They completely removed the API methods for accessing regions and I can't seem to find another way of accessing it so its possible that its an oversight on their part. 

This is what is no longer working if you happen to know something about it. Otherwise its a waiting game for the developers to update the API.

(No longer methods)
RegionManager rm = wg.getRegionManager(loc.getWorld());
`rm.getApplicableRegions(com.sk89q.worldguard.bukkit.BukkitUtil.toVector(loc));`

**Comment by RockinChaos** [comment]
Fixed; http://ci.craftationgaming.com/job/ItemJoin/137/
(Supports both WorldGuards new and old API.

**Comment by SlimeDog** [comment]
Excellent. Thanks.

**Comment by RockinChaos** [comment]
ItemJoin v4.9 has been officially released containing this bug fix for official usage.
You can grab it here; https://www.spigotmc.org/resources/itemjoin.12661/download?version=233044

Please read the changelog as the documentation has not been updated yet;
https://www.spigotmc.org/wiki/recent-changes

**Comment by SlimeDog** [comment]
Confirmed fixed in ItemJoin 4.9-b142.
